### ISSUE TITLE
code 

### ISSUE THREAD

**Issue by Dali936** [omniobject3d/OmniObject3D#40]
Hello, when will the code for the 3D perception algorithm be released?

**Comment by wutong16** [comment]
Hi, for the 3D perception part (robust point cloud recognition), we directly follow the code and pretrained models [here](https://github.com/jiawei-ren/ModelNet-C/tree/main).

**Comment by Dali936** [comment]
theanks a lot. i have anthoer question now.Sorry for my poor understanding 
![image](https://github.com/omniobject3d/OmniObject3D/assets/83868746/adb3e3d3-bb53-4816-b2bc-06a7e487410d)
in that picture 
mCE means : trained on Omniobject3d, test on Omniobject3d-c? 
mCE* means : trained on modelnet40, test on modelnet40-c?

**Comment by wutong16** [comment]
mCE means trained on ModelNet40, test on OmniObject3D-C (with the overlapped labels);
mCE* means trained on ModelNet40, test on ModelNet40-C.
OmniObject3D-C is only used as test set here.

**Comment by 944104439** [comment]
mCE* also coupled with OOD style and OOD corruption,can i understan it this way?

**Comment by Dali936** [comment]
> mCE means trained on ModelNet40, test on OmniObject3D-C (with the overlapped labels); mCE* means trained on ModelNet40, test on ModelNet40-C. OmniObject3D-C is only used as test set here.

mCE* also coupled with OOD style and OOD corruption,can i understand it this way?

**Comment by wutong16** [comment]
OOD style denotes the difference between CAD models (ModelNet40) and real-scanned models (OmniObject3D), which is not showed in ModelNet40-C. mCE* (which is tested on ModelNet40-C) only represents OOD corruption.

**Comment by Dali936** [comment]
> OOD style denotes the difference between CAD models (ModelNet40) and real-scanned models (OmniObject3D), which is not showed in ModelNet40-C. mCE* (which is tested on ModelNet40-C) only represents OOD corruption.

sorry for misquoted，what about mCE

**Comment by wutong16** [comment]
For mCE, the test set (OmniObject3D-C) involves both OOD style & OOD corruption

**Comment by Dali936** [comment]
> For mCE, the test set (OmniObject3D-C) involves both OOD style & OOD corruption

thanks for your clearly reply.
